src/phpDocumentor/Compiler/Pass/ResolveInlineLinkAndSeeTags.php

```diff
@@ -202,7 +202,7 @@ private function resolveElement(DescriptorAbstract $element, $link, $description
         $rule = $this->router->match($element);
 
         if ($rule) {
-            $url = $rule->generate($element);
+            $url = '..' . $rule->generate($element);
```

Comment:
This will probably only work in a limited number of situations; the router is intended to generate an appropriate URL, have you been able to find out why it didn't generate the right path?